akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala

```diff
@@ -9,7 +9,7 @@ import java.nio.file.{ OpenOption, Path }
 import java.util
 import java.util.concurrent.CompletionStage
 
-import akka.stream.{ javadsl, scaladsl, IOResult }
+import akka.stream.{ javadsl, scaladsl, ActorAttributes, IOResult }
```

Comment:
Why is this import needed though?

Reply:
Otherwise the link `[[ActorAttributes]]` in scala doc below doesn't work. We can, instead, use `akka.stream.ActorAttributes` below. Both way is OK for me.

Reply:
I'm afraid it'll need to be `akka.stream.ActorAttributes`, as it seems the compiler produces an 'unused import' error when it is only used in scaladoc(?)

Reply:
I was under the impression that importing it into scope just takes a red-squiggly in IntelliJ away but it still won't render in the scaladoc unless it is the fully qualified name.

Reply:
Thanks for your information, updating...

Reply:
Updated.